careers/working-at-cfpb/index.html

```diff
@@ -0,0 +1,150 @@
+{% extends "layout-side-nav.html" %}
+{% import "_vars-careers.html" as vars with context %}
+{%- set active_nav_id = 'working-at-cfpb' -%}
+
+{% block title -%}
+    Working at the CFPB
+{%- endblock %}
+
+{% block desc -%}
+    Information on careers at the CFPB.
+{%- endblock %}
+
+{% block css -%}
+    {{ super() }}
+    {# TODO: Resolve slick font issue.
+             For the slick stylesheet to pick up its font and img reference the
+             fonts and images found at /src/vendor/slick-carousel/slick/ should
+             be included in same directory as slick.css.
+             However, including the slick fonts makes the dots on the
+             /the-bureau/ page at mobile sizes enormous.
+    #}
+    <link rel="stylesheet" href="/static/css/slick.css">
+{%- endblock %}
+
+{% block content_modifiers -%}
+    {{ super() }}
+    careers-working-at-cfpb
+{%- endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }}
+    content__flush-bottom
+{%- endblock %}
+
+{% block hero -%}
+    <section class="content_hero hero">
+        <div class="hero_wrapper">
+        <div class="hero_img-container hero_img-container__small">
+            {# TODO: Replace placeholder div with image #}
+            <div class="hero_img"
+                 style="width:660px;height:300px;background-image:url({{ 'http://placehold.it/660x300.jpg' }});">
```

Comment:
I'm not sure I understand. The whole div is a placeholder and will need to be replaced with an img when the actual image arrives. Unless the replacement image will be a background on a div too? The inline width/height are just because it won't pick up the width/height from the background image dimensions.